src/routes/RescueRevisions.mjs

```diff
@@ -21,40 +38,58 @@ export default class RescueRevisions extends APIResource {
       throw new NotFoundAPIError({ parameter: 'id' })
     }
 
-    const rescueHistory = await RescueHistory.findAndCountAll({
-      where: {
-        id: ctx.params.id,
-      },
-    })
+    // const rescueHistory = await RescueHistory.findAndCountAll({
```

Comment:
![wut](https://media1.giphy.com/media/6uGhT1O4sxpi8/giphy.gif)

Reply:
Rescue history isn't fully implemented, so it now throws a `NotImplementedAPIError()`

Leaving it in as it was generated lint errors, which would fail the build.

* console.log generates warn because unexpected console.log
* removing console.log leads to unused assignments

Given the code isn't in use anyway, I thought it was better just to comment it out and throw the error instead until these endpoints are fully implemented.